.schemas-version

```diff
@@ -1 +1 @@
-v0.1.5
+v0.1.6
```

Comment:
Doesn't need to be changed here as it doesn't block this PR. It will be bumped on release.